1013-header-design-guidance.md

```diff
@@ -0,0 +1,27 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-21
+updated: 2024-10-21
+---
+
+# Header Component
+
+A page [Header](https://clarity.design/documentation/header) component defines the top of a page. Provides a consistent place for branding, navigation, search, and access to global application actions like settings and notifications.
+
+## Guidance
+
+- **Must** use one header per page and **must** be at the start of the page.
+- **Must not** use at the top of a smaller container like a popup or dialog.
```

Comment:
```suggestion
- Design teams **must not** use a header at the top of smaller containers, such as popups or dialogs, to avoid confusion and maintain a clear distinction between page-level and contextual navigation.
```